doc/API.md

```diff
@@ -480,6 +485,92 @@ netaddress string // Optional
 
 Response: standard
 
+#### /host/storage [GET]
+
+Function: Get a list of folders tracked by the host's storage manager.
+
+Parameters: none
+
+Response:
+```javascript
+{
+  "folders": [
+    {
+      "path": "/home/foo/bar",
+      "capacity": 50000000000, // bytes
+      "capacityremaining": 100000, // bytes
```

Comment:
I was thinking about this earlier. I kept them unaligned as the response is specified in JSON not go, and it doesn't seem to be a convention to align JSON fields in API docs. But aligned fields are easier to read so :+1: